scaaml/model.py

```diff
@@ -93,31 +98,40 @@ def load_model_from_disk(path, verbose=0):
 
     # load model
     mdl = load_model(path)
+    assert mdl
+    mdl = cast(Model, mdl)  # type: ignore[no-any-unimported]
 
     # display summary if requested
     if verbose:
         mdl.summary()
     return mdl
 
 
-def get_models_list(config, verbose=0):
+@dataclass
+class ModelInfo:
```

Comment:
NIT I was almost sure this would break some code as dataclasses are not subscriptable. Turns out `get_models_list` is not used.